mobile/native/src/config/mod.rs

```diff
@@ -13,10 +13,10 @@ use url::Url;
 static CONFIG: Storage<ConfigInternal> = Storage::new();
 
 #[derive(Clone)]
-struct ConfigInternal {
+pub struct ConfigInternal {
     coordinator_pubkey: PublicKey,
     esplora_endpoint: Url,
-    http_endpoint: SocketAddr,
+    pub http_endpoint: SocketAddr,
```

Comment:
🔧 Should be a getter like the others. We could even get the `coordinator_health_endpoint` directly by constructing it here.

Reply:
"like the others"? there is no getters for this struct. If I'm not adding anything to the behaviour, just adding getter for getter's sake is pointless. 

Putting the `coordinator_health_endpoint` here makes sense though, and that warrants a method certainly! thank you for your suggestion.

Reply:
I suppose getter is not the correct terminology here. What I meant is that we have free functions to access the fields in the static `CONFIG` for the other fields of `ConfigInternal`.

Reply:
the reason I didn't want to go with that, is that I do not want to expose it to the frontend, and I actually have the config struct, so there's no reason to rely on a global object to retrieve it.

Reply:
have added a dedicated getter though.